### PR TITLE
Add enzyme library and a test example

### PR DESCRIPTION
**Pull request by irikuo**
This PR adds the [`enzyme`](http://airbnb.io/enzyme/) library to simplify testing React components. It's not meant to add tests for all the components, just a initial working example of how to use it. That example is with the `ClientPortalModules` component.

### CODE REVIEW

**Comment by reevoo-samuel** [comment]
*If you're new, read the [guidelines]
(https://github.com/reevoo/guidelines/blob/master/pull_requests.md).*

Good code is **tested**, **designed for change**, **easy to understand**
and **able to cope when things go wrong**.

Good pull requests are **descriptive**, **small** and **short-lived**.

**For the Author**
- [ ] Description explains what you have done and why.
- [ ] Tests have been written (or you have explained why you don't need
tests).
- [ ] Documentation/comments have been added/updated.

**For the Reviewer**
- [ ] The code is well-designed and able to cope with change.
- [ ] The code is easy to understand.
- [ ] Tests cover happy and unhappy paths in the code.
- [ ] Tests are green on CI (or pulled/tested on your machine).

File: tests/unit/app/components/client_portal_modules.test.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import ClientPortalModules from 'app/js/components/client_portal_modules/client_portal_modules'
+
+describe('<ClientPortalModules />', () => {
+  it('renders with an empty list of modules', () => {
+    const modules = []
+    const component = shallow(<ClientPortalModules modules={modules} />)
+    expect(component.find('a').length).toBe(0)
+  })
+
+  it('renders with a single module', () => {
```
Comment (ionut998) [inline tests/unit/app/components/client_portal_modules.test.js]: do we need to check for a single module? I think only the second spec is enough.

**Comment by ionut998** [comment]
LGTM

File: tests/unit/app/components/client_portal_modules.test.js (hunk 1 above)
Comment (irikuo) [inline tests/unit/app/components/client_portal_modules.test.js]: Hmmmm, fair point... In this case doesn't make a lot of sense actually. This is just me biased by the behaviour of React Component's `this.props.children`, that is not the same with a single element or more than one. But, as I said, in this case it's true that maybe it's redundant.

**Pull request merged**